deploy-edgeX.sh

```diff
@@ -36,6 +36,10 @@ else
 	export SECURITY_IS_ON="false"
 fi
 
+docker-compose down -t 1
+docker volume prune -f
+docker-compose pull
```

Comment:
Suggest that we add a line to the startup script to make sure that we always pull down the latest nexus images -- docker will by default only check for newer images if the label is "latest."  When not using "latest" this check needs to be done manually.

Reply:
In Jenkins the VM is fresh when this is executed so it will have to pull the most recent image due to lack of docker images on the VM. This shell script is only executed once.

Reply:
Which is also to say, I think, that these are all unnecessary.

Reply:
To be removed in final version after debugging completed.